R/implicit_assignment_linter.R

```diff
@@ -45,7 +45,7 @@ implicit_assignment_linter <- function(except = c(
   "expect_output", "expect_silent",
   "local", "quo", "quos", "quote", "test_that"
 )) {
-  stopifnot(is.character(except))
+  if (!is.null(except)) stopifnot(is.character(except))
```

Comment:
I'd suggest to use `stopifnot(is.null(except) || is.character(except))` so the generated error message reflects the behavior more closely.